build/workspace.bzl

```diff
@@ -68,7 +68,7 @@ def cri_tarballs():
             name = "cri_tools_%s" % arch,
             downloaded_file_path = "cri_tools.tgz",
             sha256 = sha,
-            urls = mirror("https://github.com/kubernetes-incubator/cri-tools/releases/download/v%s/crictl-v%s-%s.tar.gz" % (CRI_TOOLS_VERSION, CRI_TOOLS_VERSION, arch)),
+            urls = mirror("https://github.com/kubernetes-sigs/cri-tools/releases/download/v%s/crictl-v%s-%s.tar.gz" % (CRI_TOOLS_VERSION, CRI_TOOLS_VERSION, arch)),
```

Comment:
@xmudrii -- One stylistic nit, which does not need to be addressed, it's just a tip for future PRs...

When you change artifact locations, try to do so as a separate commit; it makes it easier to track down, edit, or remove (if you need to later).

Reply:
@justaugustus Thank you for the tip!